app/api/api_extension.py

```diff
@@ -17,32 +26,32 @@ def ioslink():
     title="Mentorship System API",
     version="1.0",
     description="API documentation for the backend of Mentorship System. \n \n"
-    + "Mentorship System is an application that matches women in tech to mentor each other, on career development, "
-    + "through 1:1 relations during a certain period of time. \n \n"
-    + "The main repository of the Backend System can be found here: https://github.com/anitab-org/mentorship-backend"
+    + "Mentorship System is an application that matches women in tech to mentor each other, on career "
+    "development, through 1:1 relations during a certain period of time. \n \n"
+    + "The main repository of the Backend System can be found here: "
+    "https://github.com/anitab-org/mentorship-backend "
     + "\n\nThe clients for the Mentorship System: \n"
     + androidlink()
     + " | "
     + flutterlink()
     + " | "
     + ioslink()
-    + "\n\nGet started to using Backend Swagger UI here: https://github.com/anitab-org/mentorship-backend/wiki/Using-Backend-Swagger-UI"
-    + "\n\nFor more information about the project here's a link to our wiki guide: https://github.com/anitab-org/mentorship-backend/wiki"
-    + "\n\nThis <a href=https://github.com/anitab-org/mentorship-backend/blob/develop/docs/quality-assurance-test-cases.md>Quality Assurance Test cases</a> document contains examples of test scenarios to evaluate if the API is working as it should.",
+    + "\n\nGet started to using Backend Swagger UI here: "
+    "https://github.com/anitab-org/mentorship-backend/wiki/Using-Backend-Swagger-UI "
+    + "\n\nFor more information about the project here's a link to our wiki guide: "
+    "https://github.com/anitab-org/mentorship-backend/wiki "
+    + "\n\nThis <a href=https://github.com/anitab-org/mentorship-backend/blob/develop/docs/quality"
+    "-assurance-test-cases.md>Quality Assurance Test cases</a> document contains examples of test "
+    "scenarios to evaluate if the API is working as it should.",
 )
```

Comment:
same here

Reply:
Hi @epicadk, by PEP8, lines were too long. Black approved this.